engine/api/event_v2/event_run.go

```diff
@@ -2,12 +2,38 @@ package event_v2
 
 import (
 	"context"
+	"encoding/json"
 
 	"github.com/ovh/cds/engine/cache"
 	"github.com/ovh/cds/sdk"
 )
 
+func PublishRunJobRunResult(ctx context.Context, store cache.Store, eventType, vcsName, repoName string, rj sdk.V2WorkflowRunJob, rr sdk.V2WorkflowRunResult) {
+	bts, _ := json.Marshal(rr)
+	e := sdk.EventV2{
```

Comment:
in fact i'm not sure it's a good idea to share the same structure for all events...